### ISSUE TITLE
ChatSecure changes Topic to last known

### ISSUE THREAD

**Issue by ChrisDeh** [ChatSecure/ChatSecure-iOS#939]
When opening a conference in ChatSecure, the client sets the Topic to the last one he knows instead of taking the conference topic from the server. Also it is not possible to set the topic of a conference using "/topic"

**Comment by chrisballinger** [comment]
Like it tries to override the topic on the server?

**Comment by ibot87** [comment]
yes. same problem here (on the same server). if a user using chatsecure opens the app the topic of the muc is reset to the topic it had seen the last time, even if the topic has changed in the meantime. maybe it's an issue with the latest ejabberd version and chatsecure, because the server was upgraded a few weeks ago.

**Comment by kbeckmann** [comment]
Yeah I noticed this too. Just wanted to chime in and say that I can reproduce the issue. (v4.2.1)

**Comment by a-tze** [comment]
related, probably dupe of #839
